Test/GherkinFeatureFiles/display-points.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Data Point Component in WebXR Application
+  Background:
+    Given That the user is on the Meta Quest 2
+
+  Scenario: View Data Points in 3D Space
+    When the WebXR application is loaded
+    Then the data points should be rendered in 3D space
+
+  Scenario: View Data Points in 3D Space while walking
+    When the WebXR application is loaded
```

Comment:
if you want to keep all of these as When the WebXR application is loaded (which you can, then ignore my prior comment) then all you have to do is change this to say Given the WebXR application is loaded